scripts/street_sign_recognizer.py

```diff
@@ -43,20 +132,34 @@ def sign_bounding_box(self):
         (left_top, right_bottom) where left_top and right_bottom are tuples of (x_pixel, y_pixel)
             defining topleft and bottomright corners of the bounding box
         """
-        # TODO: YOUR SOLUTION HERE
-        left_top = (200, 200)
-        right_bottom = (400, 400)
-        return left_top, right_bottom
+        # Find contours based on the binary filtered image
+        contours, hierarchy = cv2.findContours(self.filt_image, cv2.RETR_TREE, \
```

Comment:
Thank you for using constants like cv2.CHAIN_APPROX_SIMPLE instead of an integer value.